docs(sources): add integrations directory with 31 source pages (ETL-980) (#739)
glassflow/clickhouse-etl: docs/app/sources/gcs/page.mdx

---
title: 'Google Cloud Storage'
description: 'Ingest objects from Google Cloud Storage into ClickHouse with GlassFlow'
---
import { Callout } from 'nextra/components'

# Google Cloud Storage

Google Cloud Storage is a scalable object storage service for storing and retrieving large amounts of data. It is commonly used as a data lake backend for analytics and machine learning.

## Using Google Cloud Storage with GlassFlow

GlassFlow can ingest data from Google Cloud Storage into ClickHouse with the same pipeline capabilities available to all GlassFlow sources — deduplication, filtering, and stateless transformations.

<Callout type="info">

> **Available on request.** Google Cloud Storage connectors are supported natively via the GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.

Reviewer: ```suggestion
**Available on request.** Google Cloud Storage connectors are natively supported in GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.
```
